Sources/AST/Token.swift

```diff
@@ -94,6 +95,7 @@ extension Token {
         case .minus, .overflowingMinus: return 20
         case .times, .overflowingTimes: return 30
         case .divide: return 30
+        case .power: return 35
         case .ampersand: return 35
```

Comment:
It looks like the precendence here should have been incremented, so as to keep the increasing sequence.

Reply:
As in make it 31?

Reply:
That, or make the ones following 40, 45, etc.